web/src/beta/lib/core/engines/Cesium/hooks.ts

```diff
@@ -740,6 +747,42 @@ export default ({
         : 1; // default as 1
   }, [property?.render?.antialias]);
 
+  // explicit rendering
+  const explicitRender = useCallback(() => {
+    const viewer = cesium.current?.cesiumElement;
+    if (!requestingRenderMode?.current || !viewer || viewer.isDestroyed()) return;
+    viewer.scene.requestRender();
+    if (requestingRenderMode.current === 1) {
+      requestingRenderMode.current = 0;
+    }
+  }, [requestingRenderMode]);
+
+  useEffect(() => {
+    const viewer = cesium.current?.cesiumElement;
+    if (!viewer || viewer.isDestroyed()) return;
+    return viewer.scene.postUpdate.addEventListener(explicitRender);
+  }, [explicitRender]);
```

Comment:
I think we should wrap the `explicitRender` function with the ref. Because `postUpdate` event is invoked in every frame, but the request render will be executed after one frame if `postUpdate` is listened depends on updating the `explicitRender` function.
```suggestion
  const explicitRenderRef = useRef();
  useEffect(() => {
    explicitRenderRef.current = explicitRender;
  }, [explicitRender]);
  useEffect(() => {
    const viewer = cesium.current?.cesiumElement;
    if (!viewer || viewer.isDestroyed()) return;
    return viewer.scene.postUpdate.addEventListener(() => {
      explicitRenderRef.current?.();
    });
  }, []);
```

Reply:
Yes it seems better this way. Thank you!